test/suites/storage_volume_import.sh

```diff
@@ -0,0 +1,44 @@
+test_storage_volume_import() {
+  truncate -s 25MiB foo.iso
+  truncate -s 25MiB foo.img
+
+  ensure_import_testimage
+
+  # importing an ISO as storage volume requires a volume name
+  ! lxc storage volume import "lxdtest-$(basename "${LXD_DIR}")" ./foo.iso || false
+  ! lxc storage volume import "lxdtest-$(basename "${LXD_DIR}")" ./foo.img --type=iso || false
+
+  # import ISO as storage volume
+  lxc storage volume import "lxdtest-$(basename "${LXD_DIR}")" ./foo.iso foo
+  lxc storage volume show "lxdtest-$(basename "${LXD_DIR}")" foo | grep -q 'content_type: iso'
```

Comment:
Would `lxc storage volume import "lxdtest-$(basename "${LXD_DIR}")" ./foo.img --type=iso bar` work? Or would it fail due to `.img`? Related to an earlier comment about filename.